src/DynamoCore/Graph/Nodes/CodeBlockNode.cs

```diff
@@ -45,6 +45,7 @@ public class CodeBlockNodeModel : NodeModel
         private readonly LibraryServices libraryServices;
 
         private bool shouldFocus = true;
+        internal ParseParam ParseParam { get; private set; }
```

Comment:
so this is a cache for keeping around results from previous compilations?

Reply:
Yes